database/migrations/2013_11_14_120127_create_zones_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('zones', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
```

Comment:
Hello, please it is parent_id and it is a foreign key!!!

Reply:
On a discuté de ça sur WhatsApp et on s'est entenu pour integer je pense bien